eagle-core/eagle-app/eagle-app-base/src/test/java/org/apache/eagle/app/stream/CEPFunctionTest.java

```diff
@@ -29,22 +30,31 @@
     public void testSiddhiFunction() {
         CEPFunction function = new CEPFunction(
             "define stream inputStream (name string, value double);\n "
-                + "from inputStream#window.timeBatch( 1 min ) \n" +
+                + "from inputStream#window.timeBatch( 10 sec ) \n" +
                 "select name, avg(value) as avgValue\n" +
                 "group by name \n" +
                 "insert into outputStream ",
             "inputStream","outputStream");
         Collector collector = new Collector() {
             @Override
             public void collect(Object key, Map event) {
-
+                Assert.assertTrue(Double.valueOf(event.get("avgValue").toString()) == 0.97);
             }
         };
         function.open(collector);
         function.transform(new HashMap<String,Object>() {{
             put("name","cpu.usage");
             put("value", 0.98);
         }});
+        function.transform(new HashMap<String,Object>() {{
+            put("name","cpu.usage");
+            put("value", 0.96);
+        }});
+        try {
+            Thread.sleep(60000);

```

Comment:
Avoid use sleep in unit test especially,here sleep for `60s`, learn to use `Semaphore` for *async unit test case*